30-64061/main.js

```diff
@@ -0,0 +1,33 @@
+// 2022 05 01 시작 - 인형뽑기
+
+function samepang(L) { // 마지막이 같은 원소가 2개면 지우는 함수. L은 쌓인 인형 리스트
+    for (i in L) {
+                
+        if ( ( (L.length) > 1 ) && (L[L.length-1] === L[L.length-2]) ) {
+            L = L.splice(L.length-2,2);
+                      
+        }
+    }
+    return L
+}
+function solution(board, moves) {
+    panglist = [];
```

Comment:
```suggestion
    const panglist = [];
```
저번에도 이 댓 남겼는데 이거 돌아가는 파일들인거 맞음?